### ISSUE TITLE
rework cli for training and classifying

### ISSUE THREAD

**Issue by gwarmstrong** [gwarmstrong/mohawk#11]
Small to-do for `classify` CLI:
- [ ] change `model` to `model_path`

Write `train` CLI:
- [ ] options for seed, number of reads to simulate, train_ratio, batch_size, gpu, summary_interval, epochs, summarize, learning_rate, log_dir, concise=True, config file
- [ ] essentially implement what is in the trial scripts
- [ ] move trial_ids and validation ids to files
- [ ] find way to test
